USAGE.md

```diff
@@ -174,3 +174,8 @@ the return code, and insert the output even if the process reports an error.
 
 Finally, the scripts directory contains some wrapper scripts that may be useful to put in your path
 and execute from the tagline.
+
+## Startup Commands
+
+At startup, de will run any Commands in `~/.de/startup`. At the moment, this is
+probably only useful for setting **TermWidth**.
```

Comment:
I actually use mine for both TermWidth and WarnAlpha (to compensate for the fact that the shade of red is different between my Mac at work and my Linux machine at home. I think the Mac does gamma correction and X doesn't..)

Theoretically, you could also use it to do some kind of shell command to do pre-processing before displaying a file (ie. you can add |gofmt to automatically have gofmt applied to any file you open, but you probably don't want to just in case you're opening something other than a go file..)